<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNetAssistant</class>
 <widget class="QMainWindow" name="QNetAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeBugAssistant</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAct"/>
   <addaction name="separator"/>
   <addaction name="tabModeAct"/>
   <addaction name="tileAct"/>
   <addaction name="separator"/>
   <addaction name="loadAct"/>
   <addaction name="saveAct"/>
   <addaction name="languageAct"/>
  </widget>
  <action name="newAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/add.png</normaloff>:/images/images/add.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="tabModeAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/tab.bmp</normaloff>:/images/images/tab.bmp</iconset>
   </property>
   <property name="text">
    <string>tab页展开</string>
   </property>
  </action>
  <action name="tileAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/tile.bmp</normaloff>:/images/images/tile.bmp</iconset>
   </property>
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="loadAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/loadconfig.bmp</normaloff>:/images/images/loadconfig.bmp</iconset>
   </property>
   <property name="text">
    <string>加载配置</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/saveconfig.bmp</normaloff>:/images/images/saveconfig.bmp</iconset>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
  </action>
  <action name="languageAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/Language.bmp</normaloff>:/images/images/Language.bmp</iconset>
   </property>
   <property name="text">
    <string>语言切换</string>
   </property>
   <property name="toolTip">
    <string>语言切换</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
